CaPPMS/Data/FaqManagerService.cs

```diff
@@ -30,10 +30,14 @@ public FaqManagerService()
                 var faqs = JsonConvert.DeserializeObject<Dictionary<Guid, FaqInformation>>(File.ReadAllText(faqDbFile.FullName));
 
                 foreach (var faq in faqs)
-
                 {
                     _ = FaqInfo.TryAdd(faq.Key, faq.Value);
-
+                    // Somehow the Key is not being set to the value in the object. This can be further explored
+                    // but this is a quick fix that allows for deletion.
+                    if (faq.Value.Guid != faq.Key)
+                    {
```

Comment:
100% I feel like this is kinda a hack solution, but I couldn't really explain why the existing GUID values weren't being saved as the key for the Faq Information, so this was the first fix that addressed already existing GUID values. Probably my next step will be to figure out what the values are when they're being saved and working my way up to figure out where the values differ. 

I found it a little weird though, because the FAQ seems to have the exact same structure as the Project Information, but the project information deletes correctly whereas the faq does not.